lemur/migrations/env.py

```diff
@@ -69,8 +69,7 @@ def run_migrations_online():
     context.configure(
         connection=connection,
         target_metadata=target_metadata,
-        **current_app.extensions["migrate"].configure_args,
-        compare_type=True
```

Comment:
Why did this need to be removed?

Reply:
we were trying to update to a newer version of sqlalchemy and this was removed to resolve a conflict in the pytest. we reverted back to sqlalchemy 1.4.0, so this has been added back.

Reply:
correction here - we are testing the new updates and getting the following error, related to the `compare_type`

```
File "/opt/lemur/lemur/manage.py", line 156, in create_all 
    stamp(revision="head")
File "/usr/local/lib/python3.9/site-packages/flask_migrate/__init__.py", line 111, in wrapped 
    f(*args, **kwargs)
File "/usr/local/lib/python3.9/site-packages/flask_migrate/__init__.py", line 259, in stamp 
    command.stamp(config, revision, sql=sql, tag=tag)
File "/usr/local/lib/python3.9/site-packages/alembic/command.py", line 678, in stamp
    script.run_env()
File "/usr/local/lib/python3.9/site-packages/alembic/script/base.py", line 582, in run_env
    util.load_python_file(self.dir, "env.py")
File "/usr/local/lib/python3.9/site-packages/alembic/util/pyfiles.py", line 94, in load_python_file
    module = load_module_py(module_id, path)
File "/usr/local/lib/python3.9/site-packages/alembic/util/pyfiles.py", line 110, in load_module_py
    spec.loader.exec_module(module) # type: ignore
File "<frozen importlib._bootstrap_external>", line 850, in exec_module
File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
File "migrations/env.py", line 86, in <module>
    run_migrations_online()
File "migrations/env.py", line 69, in run_migrations_online
    context.configure(

TypeError: alembic.context.configure() got multiple values for keyword argument 'compare_type'
```
seems like `compare_type` is being passed in via `**current_app.extensions["migrate"].configure_args` and throwing the multiple values type error.

we have tested with `compare_type` removed and everything seems to be working ok.